syncer/syncer.go

```diff
@@ -1741,7 +1759,11 @@ func (s *Syncer) handleQueryEvent(ev *replication.QueryEvent, ec eventContext) e
 			needHandleDDLs = appliedSQLs // maybe nil
 		}
 
-		job := newDDLJob(nil, needHandleDDLs, *ec.lastPos, *ec.currentPos, nil, nil, *ec.traceID)
+		if err := s.flushJobs(); err != nil {
+			return err
+		}
+
+		job := newDDLJob(nil, needHandleDDLs, *ec.lastPos, *ec.currentPos, nil, nil, *ec.traceID, sourceTbls)
```

Comment:
Is there any problem to use `sourceTbls` here? see <https://github.com/pingcap/tidb-enterprise-tools/pull/471> 😂.